lib/connections/relay_listen.go

```diff
@@ -95,7 +98,7 @@ func (t *relayListener) Serve() {
 			err = tlsTimedHandshake(tc)
 			if err != nil {
 				tc.Close()
-				l.Infoln("TLS handshake (BEP/relay):", err)
+				l.Debugln("Listen (BEP/relay): TLS handshake:", err)
```

Comment:
Same here, only place that reports listen errors, so not cool to hide it.

Reply:
I will make everything log on info level that results in an early return/continue/break, does that sound sensible?

Reply:
Well I think I pointed out the relevant places in this change, but yeah, anything that doesn't return an error should probably be logged.